pkg/schedule/schedulers/evict_slow_trend.go

```diff
@@ -322,18 +327,17 @@ func chooseEvictCandidate(cluster schedule.Cluster) (slowStore *core.StoreInfo)
 	}
 
 	storeSlowTrendActionStatusGauge.WithLabelValues("cand.add").Inc()
-	log.Info("evict-slow-trend-scheduler canptured candidate", zap.Uint64("store-id", store.GetID()))
+	log.Info("evict-slow-trend-scheduler captured candidate", zap.Uint64("store-id", store.GetID()))
 	return store
 }
 
-func checkStoresAreUpdated(cluster schedule.Cluster, baseline *core.StoreInfo) bool {
+func checkStoresAreUpdated(cluster schedule.Cluster, slowStoreID uint64, baselineTS time.Time) bool {
```

Comment:
where the `baselineTS` is used in this function?

Reply:
The param `baselineTS` is just kept same with previous naming, and used in the following judgement at line 350:
https://github.com/tikv/pd/blob/3779808515edb6362d8f7d28980afca4af687a62/pkg/schedule/schedulers/evict_slow_trend.go#L350

Reply:
Btw， the ts has different name such as `baselineTs`,`slowStoreRecordTS`,`captureTS`,  it's a little strange.

Reply:
`captureTS` represents the TS when the first slowStore has been captured, it's a prerequisite for later processing.

How about renaming `baselineTS` with `slowStoreCapturedTS` ?

Reply:
how about `slowStoreRecordTS` ?